aggregator/data.go

```diff
@@ -126,16 +126,8 @@ func (a *Aggregator) processk8s() {
 			a.processPod(d)
 		case k8s.SERVICE:
 			a.processSvc(d)
-		case k8s.REPLICASET:
-			a.processReplicaSet(d)
-		case k8s.DEPLOYMENT:
-			a.processDeployment(d)
-		case k8s.ENDPOINTS:
-			a.processEndpoints(d)
 		case k8s.CONTAINER:
 			a.processContainer(d)
-		case k8s.DAEMONSET:
-			a.processDaemonSet(d)
 		default:
```

Comment:
I guess this will now log a lot of warnings, since you're removing a lot of cases?

Reply:
No, we aren't watching all of these removed resource types anyway, so we won't end up sending messages to the channel that this function is pulling from, so we shouldn't see these logs.

Reply:
OH yeah, rereading this it makes total sense :+1: